tools/releaseBuild/azureDevOps/releaseBuild.yml

```diff
@@ -168,74 +122,229 @@ stages:
         parameters:
           Architecture: fxdependentWinDesktop
 
-      - template: templates/windows-packaging.yml
-        parameters:
-          Architecture: x64
-          parentJob: build_windows_x64_release
-
-      - template: templates/windows-packaging.yml
-        parameters:
-          Architecture: x64
-          BuildConfiguration: minSize
-          parentJob: build_windows_x64_minSize
-
-      - template: templates/windows-packaging.yml
-        parameters:
-          Architecture: x86
-          parentJob: build_windows_x86_release
-
-      - template: templates/windows-packaging.yml
-        parameters:
-          Architecture: arm
-          parentJob: build_windows_arm_release
-
-      - template: templates/windows-packaging.yml
-        parameters:
-          Architecture: arm64
-          parentJob: build_windows_arm64_release
-
-      - template: templates/windows-packaging.yml
-        parameters:
-          Architecture: fxdependent
-          parentJob: build_windows_fxdependent_release
+  - stage: SignFiles
+    displayName: Sign files
+    dependsOn: ['windows', 'linux', 'macos']
+    jobs:
+    - template: templates/mac-file-signing.yml
+      parameters:
+        buildArchitecture: x64
+
+    - template: templates/mac-file-signing.yml
+      parameters:
+        buildArchitecture: arm64
+
+    - job: SignFilesWinLinux
+      pool:
+        name: $(windowsPool)
+        demands:
+        - ImageOverride -equals PSMMS2019-Secure
+      displayName: Sign files
+
+      variables:
+      - group: ESRP
+      - name: runCodesignValidationInjection
+        value: false
+      - name: NugetSecurityAnalysisWarningLevel
+        value: none
+      - name: repoFolder
+        value: PowerShell
+      - name: repoRoot
+        value: $(Agent.BuildDirectory)\$(repoFolder)
+      - name: complianceRepoFolder
+        value: compliance
+
+      strategy:
+        matrix:
+          linux-x64:
```

Comment:
We should consider dynamically producing the matrix, so we can move it into a single JSON in our repo as we need this matrix in more places.

Reply:
I can do that as part of the next iteration, i need to split the deb build and packaging.